README.md

```
@@ -290,6 +292,9 @@ Found a bug or need help?
 
 - **Issues**: [Create an issue](https://github.com/avni-7span/touch_craft_editor/issues)
 
+# **👥 Maintainers**
+
+  - [Avni Prajapati](https://github.com/avni-7span)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)